app/scripts/cartodb/cartodb-api-config.js

```diff
@@ -27,6 +27,11 @@
                 avgEnergyStar: 59,
                 ghgBuildings: 60,
                 numBuildings: 1879
+            },
+            2015: {
```

Comment:
Right, that is to be addressed with #213. Intention is for us to push the 2015 update before working to make it more easily updated.